src/main/java/bio/terra/service/auth/iam/sam/SamIam.java

```diff
@@ -216,9 +216,12 @@ CreateResourceRequestV2 createDatasetResourceRequest(
 
     CreateResourceRequestV2 req = new CreateResourceRequestV2().resourceId(datasetId.toString());
 
-    req.putPoliciesItem(
-        IamRole.ADMIN.toString(),
-        createAccessPolicyOne(IamRole.ADMIN, samConfig.adminsGroupEmail()));
+    Optional.of(samConfig.adminsGroupEmail())
+        .map(
```

Comment:
Instead of `map()` I would use `ifPresent` here since the result of this operation is ignored.
```suggestion
        .ifPresent(
```